docs/reference/Server/Curses/Curses/index.html

```diff
@@ -141,28 +139,28 @@
 </b>
 <p>
 <div class=slotDescription>
-Inserts the string at the current position on the terminal, 
+Inserts the string at the current position on the terminal,
 	pushing existing text to the right. Returns self.
 </div>
 <a name="Curses-move"></a><b>
 move(x, y)
 </b>
 <p>
 <div class=slotDescription>
-Moves the cursor to column x and row y on the terminal. 
-	(0, 0) is at the top-left of the terminal. Returns self. 
-	Be careful if you are used to the C implementation of Curses 
-	as the coordinates are the other way around. Here they are like 
-	graphical XY coordinates, whereas in C Curses the rational is 
-	more like typing text. You go down on the line you want and then 
-	you move horizontally.
+Moves the cursor to column x and row y on the terminal.
+	(0, 0) is at the top-left of the terminal. Returns self.
+	- Be careful if you are used to the C implementation of Curses
+	- as the coordinates are the other way around. Here they are like
+	- graphical XY coordinates, whereas in C Curses the rational is
+	- more like typing text. You go down on the line you want and then
+	- you move horizontally.
```

Comment:
Ah, this needs a change I pushed after #342 was already merged.  Mea culpa, I'll add it to this PR.